packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -43,7 +43,7 @@ You can also [configure a custom email notification domain](#about-custom-email-
 
 ### Caveats
 
-* Most custom domains are configured using one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain). However, there are no restrictions on using any other domain such as your root domain, if that fits your use case for your sign-in page taking over your main website.
+* You must provide one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain) for customizing the Okta URL domain. Okta doesn't support the use of a root name.
```

Comment:
Maybe another caveat can be added:  “The DNS TXT and CNAME record name and values provided to you in your domain configuration must be resolvable and contain the values provided by Okta.  This can be validated with a DNS query tool such as ‘[dig](https://en.wikipedia.org/wiki/Dig_(command))’”